text2gremlin/AST_Text2Gremlin/base/test/combination_explosion_report.md

```
@@ -0,0 +1,129 @@
+| 2 | `hasLabel()` | `'forum'` | ✅ 可替换为任意顶点标签 | 5 |
+| 3 | `out()` | `'contain'` | ✅ 可替换为任意边标签 | 8 |
+| 4 | `hasLabel()` | `'user'` | ✅ 可替换为任意顶点标签 | 5 |
+| 5 | `out()` | `'likes'` | ✅ 可替换为任意边标签 | 8 |
+| 6 | `hasLabel()` | `'post'` | ✅ 可替换为任意顶点标签 | 5 |
+| 7 | `out()` | `'has'` | ✅ 可替换为任意边标签 | 8 |
+| 8 | `hasLabel()` | `'tag'` | ✅ 可替换为任意顶点标签 | 5 |
+
+## 🧮 组合数学计算
+
+### 理论最大组合数
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)